index.html

```diff
@@ -3211,7 +3211,22 @@ <h3>Reserved Extension Points</h3>
               <th>Description</th>
             </tr>
           </thead>
+
           <tbody>
+            <tr>
+              <td>`confidenceMethod`</td>
+              <td>
+A property used for specifying how a <a>verifier</a> can gain more confidence
+in, the presentation of the credential being associated with a particular
```

Comment:
two suggestions. First, the comma after in is not needed. Second, 'being associated with a particular subject' is either ambiguous or unnecessary. All credentials are associated with a subject as they have a subject property, so does this mean that the presenter is associated with a particular subject or not?
I suggest simply remove 'being associated with a particular subject' or clarify to 'can gain more confidence
that the presenter of the credential is associated with the subject"

Reply:
Definitely, remove the extraneous comma.
```suggestion
in the presentation of the credential being associated with a particular
```